WordPress/Classes/ViewRelated/NUX/SigninEditingState.swift

```diff
@@ -1,6 +1,6 @@
 import Foundation
 
 class SigninEditingState {
-    static var signinEditingStateActive = false
+    static var signinEditingStateActive = true
```

Comment:
I went back and retested in the simulator with regular animations, slow animation, and then on a device for good measure.  I think what I was seeing before, super jumpy semi-glitchy keyboard animations was just the simulator being a pita (sorry for the noise!).  However, something I'm still seeing is this: 

![simulator screen shot jul 12 2017 1 12 48 pm](https://user-images.githubusercontent.com/1435271/28133569-163fe50a-6706-11e7-9593-0a0caf323eb4.png)

Notice how the keyboard is visible but the view has not adjusted its bottom constraint. So, I'll amend my earlier request to just needing to get the view properly sizing.